compiler/cpp/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ set(compiler_core
 set(thrift-compiler_SOURCES
     src/thrift/main.cc
     src/thrift/audit/t_audit.cpp
+    ${compiler_core}
```

Comment:
This will still break the thrift compiler plug-in module support, won't it?  The compiler_core modules must be conditionally added depending on the WITH_PLUGIN setting.